2022-02-18 13:26:36,339 INFO    MainThread:16703 [wandb_setup.py:_flush():71] setting env: {}
2022-02-18 13:26:36,339 INFO    MainThread:16703 [wandb_setup.py:_flush():71] setting login settings: {}
2022-02-18 13:26:36,339 INFO    MainThread:16703 [wandb_init.py:_log_setup():371] Logging user logs to /home/simon/CDE_UBS/thesis/testing/wandb/run-20220218_132636-1ooxtcb7/logs/debug.log
2022-02-18 13:26:36,340 INFO    MainThread:16703 [wandb_init.py:_log_setup():372] Logging internal logs to /home/simon/CDE_UBS/thesis/testing/wandb/run-20220218_132636-1ooxtcb7/logs/debug-internal.log
2022-02-18 13:26:36,340 INFO    MainThread:16703 [wandb_init.py:_jupyter_setup():321] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7f9769cf4cd0>
2022-02-18 13:26:36,340 INFO    MainThread:16703 [wandb_init.py:init():404] calling init triggers
2022-02-18 13:26:36,340 INFO    MainThread:16703 [wandb_init.py:init():411] wandb.init called with sweep_config: {}
config: {}
2022-02-18 13:26:36,340 INFO    MainThread:16703 [wandb_init.py:init():460] starting backend
2022-02-18 13:26:36,341 INFO    MainThread:16703 [backend.py:_multiprocessing_setup():101] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-02-18 13:26:36,383 INFO    MainThread:16703 [backend.py:ensure_launched():216] starting backend process...
2022-02-18 13:26:36,432 INFO    MainThread:16703 [backend.py:ensure_launched():222] started backend process with pid: 16747
2022-02-18 13:26:36,434 INFO    MainThread:16703 [wandb_init.py:init():469] backend started and connected
2022-02-18 13:26:36,480 INFO    MainThread:16703 [wandb_run.py:_label_probe_notebook():911] probe notebook
2022-02-18 13:26:36,481 INFO    MainThread:16703 [wandb_run.py:_label_probe_notebook():921] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2022-02-18 13:26:36,481 INFO    MainThread:16703 [wandb_init.py:init():533] updated telemetry
2022-02-18 13:26:36,589 INFO    MainThread:16703 [wandb_init.py:init():563] communicating current version
2022-02-18 13:26:38,416 INFO    MainThread:16703 [wandb_init.py:init():568] got version response upgrade_message: "wandb version 0.12.10 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-02-18 13:26:38,417 INFO    MainThread:16703 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-02-18 13:26:38,642 INFO    MainThread:16703 [wandb_init.py:init():606] starting run threads in backend
2022-02-18 13:26:43,652 INFO    MainThread:16703 [wandb_run.py:_console_start():1810] atexit reg
2022-02-18 13:26:43,653 INFO    MainThread:16703 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.WRAP
2022-02-18 13:26:43,653 INFO    MainThread:16703 [wandb_run.py:_redirect():1721] Wrapping output streams.
2022-02-18 13:26:43,654 INFO    MainThread:16703 [wandb_run.py:_redirect():1745] Redirects installed.
2022-02-18 13:26:43,654 INFO    MainThread:16703 [wandb_init.py:init():633] run started, returning control to user process
2022-02-18 13:39:10,004 INFO    MainThread:16703 [wandb_init.py:_pause_backend():284] pausing backend
2022-02-18 13:39:10,004 INFO    MainThread:16703 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-02-18 14:18:01,535 INFO    MainThread:16703 [wandb_init.py:_resume_backend():293] resuming backend
2022-02-18 14:18:01,565 INFO    MainThread:16703 [wandb_setup.py:_flush():71] setting env: {}
2022-02-18 14:18:01,565 INFO    MainThread:16703 [wandb_setup.py:_flush():71] setting login settings: {}
2022-02-18 14:18:01,565 INFO    MainThread:16703 [wandb_init.py:_log_setup():371] Logging user logs to /home/simon/CDE_UBS/thesis/testing/wandb/run-20220218_141801-51js22it/logs/debug.log
2022-02-18 14:18:01,565 INFO    MainThread:16703 [wandb_init.py:_log_setup():372] Logging internal logs to /home/simon/CDE_UBS/thesis/testing/wandb/run-20220218_141801-51js22it/logs/debug-internal.log
2022-02-18 14:18:01,566 INFO    MainThread:16703 [wandb_init.py:init():404] calling init triggers
2022-02-18 14:18:01,566 INFO    MainThread:16703 [wandb_init.py:init():411] wandb.init called with sweep_config: {}
config: {}
2022-02-18 14:18:01,566 INFO    MainThread:16703 [wandb_init.py:init():426] re-initializing run, found existing run on stack: 1ooxtcb7
2022-02-18 14:18:01,568 INFO    MainThread:16703 [wandb_run.py:finish():1457] finishing run simon-donike/SRCNN/1ooxtcb7
2022-02-18 14:18:01,569 INFO    MainThread:16703 [jupyter.py:save_history():429] not saving jupyter history
2022-02-18 14:18:01,570 INFO    MainThread:16703 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-02-18 14:18:01,570 INFO    MainThread:16703 [wandb_init.py:_jupyter_teardown():303] cleaning up jupyter logic
2022-02-18 14:18:01,570 INFO    MainThread:16703 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 0
2022-02-18 14:18:01,571 INFO    MainThread:16703 [wandb_run.py:_restore():1752] restore
2022-02-18 14:18:02,135 INFO    MainThread:16703 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 761
  total_bytes: 761
}

2022-02-18 14:18:02,475 INFO    MainThread:16703 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 761
  total_bytes: 761
}

2022-02-18 14:18:03,359 INFO    MainThread:16703 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 761
  total_bytes: 10074
}

2022-02-18 14:18:03,463 INFO    MainThread:16703 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 761
  total_bytes: 25266
}

2022-02-18 14:18:03,567 INFO    MainThread:16703 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 761
  total_bytes: 25266
}

2022-02-18 14:18:03,670 INFO    MainThread:16703 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 761
  total_bytes: 25266
}

2022-02-18 14:18:03,772 INFO    MainThread:16703 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 25266
  total_bytes: 25266
}

2022-02-18 14:18:03,876 INFO    MainThread:16703 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 25266
  total_bytes: 25266
}

2022-02-18 14:18:03,979 INFO    MainThread:16703 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 25266
  total_bytes: 25266
}

2022-02-18 14:18:04,081 INFO    MainThread:16703 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 25266
  total_bytes: 25266
}

2022-02-18 14:18:04,184 INFO    MainThread:16703 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 25266
  total_bytes: 25266
}

2022-02-18 14:18:04,976 INFO    MainThread:16703 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 25266
  total_bytes: 25266
}

2022-02-18 14:18:05,497 INFO    MainThread:16703 [wandb_run.py:_wait_for_finish():1912] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 25266
  total_bytes: 25266
}
local_info {
}

2022-02-18 14:18:07,863 INFO    MainThread:16703 [wandb_run.py:_append_history():2130] rendering history
2022-02-18 14:18:07,864 INFO    MainThread:16703 [wandb_run.py:_append_summary():2085] rendering summary
2022-02-18 14:18:07,864 INFO    MainThread:16703 [wandb_run.py:_append_files():2180] logging synced files
